### DISCUSSION TITLE
VSCode settings seem out of date

### DISCUSSION THREAD

**Discussion by RJ-Eckie** [SynthstromAudible/DelugeFirmware#2864] [General]
In c_cpp_properties.json there are two references to toolchain v13, and another one in settings.json. I started running into some building issues, and ultimately re-cloning the entire repo and changing this to v16 fixed it. I think someone who is more comfortable with these than me should probably have a look at them!

**Comment by m-m-adams** [comment]
I don't use vscode all that much but they probably just need to be bumped. Afaik those should only affect linting though since the build is controlled by cmake
